tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -883,9 +883,9 @@ pc/test_lag_2.py::test_lag_db_status_with_po_update:
 
 pc/test_lag_member.py:
   skip:
-    reason: "Not support dualtor topo"
+    reason: "Not support dualtor or t0 backend topo"
     conditions:
-        - "'dualtor' in topo_name"
+        - "topo_name in ['t0-backend', 'dualtor']"
```

Comment:
skip will not work for topologies like 'dualtor-aa'

Reply:
done